### DISCUSSION TITLE
Configuration R7800

### DISCUSSION THREAD

**Discussion by JosephGilles** [Ysurac/openmptcprouter#2738] [General]
Bonjour à tous !
Je tente de configurer mon routeur R7800 (flashé avec openmptctouter).
La mise en place du firmware s’est passée sans problème, par contre je n’arrive pas à configurer les ports physiques.
Voici mon un schéma simplifier de mon installation (voir ci-dessous).
Mon but est de pouvoir utiliser ma BBOX2 via la passerelle 192.168.0.1 et le R7800 (agrégation) via la passerelle 192.168.0.102
Pourriez vous m’aider à configurer les ports WAN1 & WAN2 dans le R7800 ?
Merci d'avance pour votre aide :-)
Joseph

PS : Le DHCP est désactivé sur les 3 routeurs (R7800 & BBOX & MR600), et activé sur RBK852 (mon réseau local).

![image001](https://user-images.githubusercontent.com/67262096/213925829-f2a29b5b-6d2c-4338-8786-a044fa92954b.png)

<img width="743" alt="Capture d’écran 2023-01-22 à 16 58 51" src="https://user-images.githubusercontent.com/67262096/213925793-5695849c-1299-4b81-b2f2-adaf79a18350.png">

<img width="1246" alt="Capture d’écran 2023-01-22 à 16 57 27" src="https://user-images.githubusercontent.com/67262096/213925734-f48abcf6-4282-4a90-9104-e16b4a03be19.png">
hubusercontent.com/67262096/213925396-9c7f3407-6ccf-474e-bd4e-06d7992819b8.png)

<img width="1220" alt="Capture d’écran 2023-01-22 à 16 58 19" src="https://user-images.githubusercontent.com/67262096/213925776-b0929f67-7712-4267-a6ba-e273d81ca8f3.png">

<img width="1078" alt="Capture d’écran 2023-01-22 à 17 06 09" src="https://user-images.githubusercontent.com/67262096/213926146-f70f4ea7-b088-4b0f-a96d-b285a9742e99.png">

**Comment by iambenmitchell** [comment]
Here is my setup: 

Port 1 = Raspberry Pi (OMR)
Port 2 - 6 = LAN (VLAN 10)
Port 7 = WAN 1 (VLAN 70)
Port 8 = WAN 2 (VLAN 80)

![image](https://user-images.githubusercontent.com/27426792/213926785-84ab56d6-291e-4c08-a10a-8840bc3b7aa3.jpeg)
![image](https://user-images.githubusercontent.com/27426792/213926788-f2524bcf-7757-484d-bde9-a9320a183c7c.jpeg)

**Reply by iambenmitchell** [reply]
In OMR I don't have my LAN interface assigned to a VLAN. But you do. Perhaps that is your issue? ![image](https://user-images.githubusercontent.com/27426792/213926923-2804f54c-ff1b-48a3-96ab-cc0be8b9bc3f.jpeg)

**Comment by JosephGilles** [comment]
Thank you for your reply !
I don't see where to assign the physical ports (1,2,3,4) to a vlan ...
thé in advance
Joseph

**Reply by iambenmitchell** [reply]
It should be called 'PVID' or something like that.
